lib/manageiq/appliance_console/database_admin.rb

```diff
@@ -43,6 +43,11 @@ def object_store_backup?
 
       def ask_questions
         setting_header
+        if action == :restore && LinuxAdmin::Service.new("evmserverd").running?
+          say("\nDatabase #{action} failed. Please execute \“Stop EVM Server Processes\” command and try again.")
```

Comment:
Can't action only be restore here?

Reply:
Also I think better grammar would be `Please execute the "Stop EVM Server Processes" command and try again`

Reply:
```actions```: could be ```:dump``` here

Reply:
Not according to the conditional ...

Reply:
@carbonin by here I meant inside ```#ask_questions```,
 Is your question about changing ```#{action}``` to "restore"

Reply:
> Is your question about changing #{action} to "restore"

Yes